flake8_quotes/__init__.py

```diff
@@ -68,16 +90,20 @@ def get_quotes_errors(self, file_contents):
                 # ignore non strings
                 continue
 
-            if not token.string.startswith(self.quotes['bad_single']):
-                # ignore strings that do not start with our quotes
+            if token.string[0:3] in self.MULTILINE_QUOTES_STRINGS:
+                if self.multiline_quotes is None or token.string.startswith(self.multiline_quotes['bad']):
+                    continue
+
+            if not token.string.startswith(self.quotes['bad']):
+                # ignore strings that do not start with our quote
                 continue
 
-            if token.string.startswith(self.quotes['bad_multiline']):
-                # ignore multiline strings
+            if self.quotes['good'] in token.string:
+                # ignore quotes wrapped in our quotes (e.g. `'` in `"it's"`)
                 continue
 
-            if self.quotes['good_single'] in token.string:
-                # ignore alternate quotes wrapped in our quotes (e.g. `'` in `"it's"`)
+            if self.multiline_quotes['good'] in token.string:
+                # ignore multiline quotes wrapped in our quotes
                 continue
```

Comment:
We prob don't need to exempt strings containing multiline quotes (e.g. `"""` in `'its"""'`)